### ISSUE TITLE
[Feature Question] Colored UI items?

### ISSUE THREAD

**Issue by wpferguson** [darktable-org/darktable#7682]
When I was rewriting script_manager I changed the button labels from Enable/Disable <script> to <script> stopped/started.  At the time I thought that colored buttons might be a better way to communicate state, i.e. red for stopped, green for running.  So today I decided to RTFGTKRM (read the !*#@ GTK Reference Manual) and figured out how to name lua widgets.  Then I tweaked script_manager, added a couple of CSS overrides and ended up with:

![colorbuttons](https://user-images.githubusercontent.com/1634891/103571723-68ca8e00-4e99-11eb-9c1f-b59c8cf4cba3.jpg)

I'm not sure how this fits into the user interface guidelines, or if it fits at all.  Is this a good idea or not?  Should I just quietly drop this?

**Comment by wpferguson** [comment]
I had another thought about this.  In order to have colored UI items, someone has to add the CSS, so it could be up to the user.  As I look at the picture, I find the colors distracting so I probably wont.

**Comment by Nilvus** [comment]
> I had another thought about this. In order to have colored UI items, someone has to add the CSS, so it could be up to the user. As I look at the picture, I find the colors distracting so I probably wont.

That's exactly why I would be against added colors in the UI. We have improve the UI to let more space to the image and avoid any distracting things, to let the eyes of the user go in the image more than anything else.

Of course, colors could help see things but we don't want distracting things in the UI.

**Comment by TurboGit** [comment]
Yes, I 100% agree with @Nilvus, dt has always refrained from using color in UI to avoid distracting users and I think this is a good thing. In the UI we are just using bold to highlight/show enabled vs disabled state.

**Comment by wpferguson** [comment]
Then let's just close this.

**Comment by johnny-bit** [comment]
Colors aren't bad, just needs to make sure that the colors aren't jarring, distracting and over large areas and it'll be fine imo.

**Comment by Nilvus** [comment]
> Colors aren't bad, just needs to make sure that the colors aren't jarring, distracting and over large areas and it'll be fine imo.

Of course! And color labels or sliders with colors are the best example.

**Comment by johnny-bit** [comment]
Color can convey info

So I'd propose instead of whole button go something like this:

:red_circle:  adjust time
:green_circle:  post sharpen

and it'll be fine if you want to use colors ;) additionally cthat can be combined with :stop_button: or :play_or_pause_button: kinds of symbols

**Comment by wpferguson** [comment]
@johnny-bit I actually like that.  Any idea how I would do that with CSS?

**Comment by wpferguson** [comment]
Since I only have the button label, I tried the ::first-character pseudo element but that's not supported.  Next I tried getting rid of started/stopped and just setting the labels to a not too bright shade of red and green (210,0,0 and 0,210,0).

![colorlabels](https://user-images.githubusercontent.com/1634891/103602287-a2bb8480-4ed9-11eb-9e76-7fe03082e29e.jpg)

This one I could live with.  I don't leave script_manager open most of the time, but even if I did this wouldn't be too bad.
